### PR TITLE
Fixed typo in event name

### LINKED ISSUE #18: Constant admob.EVENT_STATUS_AUTORIZED is misspelled

### PR DESCRIPTION
**Pull request by britzl**
Fixes #18

### CODE REVIEW

**Comment by AGulev** [comment]
this is not so small fix. it will break all the games with admob integrated.
We should make an announcement, or keep duplicate of the constant with the same number on Lua side (not in enum)

**Comment by britzl** [comment]
Good point. Maybe better to keep a duplicate then.

File: extension-admob/src/admob.cpp [hunk 1]
```diff
@@ -206,7 +206,7 @@ static void LuaInit(lua_State* L)
     SETCONSTANT(EVENT_DESTROYED)
     SETCONSTANT(EVENT_JSON_ERROR)
     SETCONSTANT(EVENT_IMPRESSION_RECORDED)
-    SETCONSTANT(EVENT_STATUS_AUTORIZED)
```
Comment (AGulev) [inline extension-admob/src/admob.cpp:209]: I mean to keep `EVENT_STATUS_AUTORIZED` here, but add it manually without SETCONSTANT macros with the same value (13)

Comment (britzl) [reply extension-admob/src/admob.cpp:209]: Yep, I did this change now.

**Review by AGulev — APPROVED** [review]

**Pull request merged**